drivers/staging/vc04_services/interface/vchiq_arm/vchiq_core.c

```diff
@@ -544,13 +546,15 @@ remote_event_signal_local(wait_queue_head_t *wq, struct remote_event *event)
 	wake_up_all(wq);
 }
 
+/* FIXME */
 static inline void
 remote_event_poll(wait_queue_head_t *wq, struct remote_event *event)
 {
 	if (event->fired && event->armed)
 		remote_event_signal_local(wq, event);
 }
 
+/* FIXME */
```

Comment:
VCHIQ used a small, fixed number of remote events. It is simplest to enumerate them here for polling.